react/src/components/Projects/ProjectListing.tsx

```diff
@@ -43,24 +51,33 @@ export const ProjectListing: React.FC = () => {
         </thead>
         <tbody>
           {data?.map((proj) => (
-            <tr key={proj.id} onClick={() => navigateToProject(proj.uuid)}>
-              <td>{proj.name}</td>
-              <td>
+            <tr key={proj.id}>
+              <td onClick={() => navigateToProject(proj.uuid)}>{proj.name}</td>
+              <td onClick={() => navigateToProject(proj.uuid)}>
                 {proj.ds_project?.value.projectId}{' '}
                 {proj.ds_project?.value.title}
               </td>
               <td>
                 <Button>
                   <Icon name="edit-document"></Icon>
                 </Button>
-                <Button>
+                <Button onClick={() => toggleDeleteModal(proj)}>
                   <Icon name="trash"></Icon>
                 </Button>
               </td>
             </tr>
           ))}
         </tbody>
       </table>
+
+      {selectedProjectForDeletion && (
+        <DeleteMapModal
+          isOpen={!!selectedProjectForDeletion}
+          toggle={() => toggleDeleteModal(null)}
```

Comment:
`toggleDeleteModal` can be removed and just `setSelectedProjectForDeletion` used.

```suggestion
                <Button onClick={() => setSelectedProjectForDeletion(proj)}>
                  <Icon name="trash"></Icon>
                </Button>
              </td>
            </tr>
          ))}
        </tbody>
      </table>

      {selectedProjectForDeletion && (
        <DeleteMapModal
          isOpen={!!selectedProjectForDeletion}
          toggle={() => setSelectedProjectForDeletion(null)}
```

Reply:
Makes way more sense. Thanks.